umh.docs.umh.app/content/en/docs/production-guide/administration/normalize-kafka-topics.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Normalize Kafka Topics"
+content_type: task
+description: |
+    This page describes how to reduce the amount of Kafka Topics in order to
+    lower the overhead.
+weight: 50
+edition: community
+---
+
+<!-- overview -->
+
+Kafka excels at processing a high volume of messages but can encounter difficulties
```

Comment:
maybe also mention MQTT and Kafka. Because the only reason why there are so many topics are because of how MQTT works.